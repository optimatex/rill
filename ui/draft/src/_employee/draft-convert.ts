export { convertToHTML, convertFromHTML } from 'draft-convert';
