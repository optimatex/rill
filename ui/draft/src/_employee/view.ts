export {
  React,
  useState,
  useCallback,
  css,
  styled,
  createGlobalStyle,
} from '@rill/view';
