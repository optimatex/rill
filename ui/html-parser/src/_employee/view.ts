export { React, RC } from '@rill/view';
