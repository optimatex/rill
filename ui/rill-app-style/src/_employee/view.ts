export { createGlobalStyle, css } from '@rill/view';
