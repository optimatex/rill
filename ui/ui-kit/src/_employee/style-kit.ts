export {
  // appLayout
  appLayoutCss,
  screenCss,
  sidebarColumnCss,
  flatColumnCss,
  mainColumnCss,
  // layout
  listCss,
  // Brand
  logCss,
  dualLogoCss,
  // Controller
  trivialAppHeaderCss,
  // _menu
  menuWrapCss,
  menuItemCss,
  // Image
  imageCss,
  imagesListCss,
  imagesGridCss,
  imagesLayoutItemCss,
  imageCropperCss,
  // Tags
  tag,
  tagsList,
  // Ui-primal
  // _grid
  gridRowContainerCss,
  gridrillRowContainerCss,
  gridColumnCss,
  // _card
  cardContainerCss,
  postContainerCss,
  postTitleCss,
  panelCss,
  // _buttons
  buttonCss,
  buttonActiveCss,
  buticonCss,
  // _form
  formContainerCss,
  formFieldFlatCss,
  formFieldColumnCss,
  formFieldErrorCss,
  formLabelTextCss,
  inputCss,
  toggleContainerCss,
  toggleContainerActiveCss,
  toggleRoundCss,
  toggleRoundActiveCss,
  // _article
  paragraphCss,
  nameCss,
  richContentWrapCss,
  // _table
  tableContainerCss,
  tableHeadCss,
  tableRowCss,
  tableColumnCss,
  // Unknown
  // _object
  stringValueCss,
  numberValueCss,
  booleanValueCss,
  emptyValueCss,
  arrayWrapCss,
  arrayItemCss,
  objectKeyCss,
  objectWrapCss,
  objectPairCss,
  // Unsorted
  lightIndicatorCss,
  popoverCss,
  pipeCss,
  hintableLogCss,
  fixedPanelCss,
} from '@rill/style-kit';
