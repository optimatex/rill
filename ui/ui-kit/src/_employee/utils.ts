export {
  toPairs,
  find,
  has,
  path,
  prop,
  curry,
  assoc,
  lensProp,
  view,
  set,
} from '@rill/utils';
