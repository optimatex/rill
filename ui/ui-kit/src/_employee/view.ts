export {
  React,
  useState,
  useCallback,
  // styled-components
  styled,
  // Lens
  // getTestId,
  // Types
  FC,
  RC,
  ReactNode,
  LegacyRef,
  // PropTypes
  IContainerProps,
  // Event types
  FormEvent,
  MouseEvent,
} from '@rill/view';
