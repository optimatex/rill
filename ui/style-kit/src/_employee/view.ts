import { css as originalCss } from '@rill/view';

export const css = originalCss;
