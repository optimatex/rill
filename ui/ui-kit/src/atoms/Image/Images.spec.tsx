import { render, mockImages } from 'test';
import { React } from '_employee/view';

import { ImagesLayout } from '.';

describe('ImagesLayout', () => {
  it('should render without errors', () => {
    const { getByTestId } = render(<ImagesLayout images={mockImages} />);
    const el = getByTestId('images-layout');

    expect(el).toMatchSnapshot();
  });
});
