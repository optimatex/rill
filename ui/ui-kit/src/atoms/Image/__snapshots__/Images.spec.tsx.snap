// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ImagesLayout should render without errors 1`] = `
.c1 {
  object-fit: cover;
  max-width: 100%;
  display: block;
}

.c0 {
  grid-column-start: auto;
  grid-row-start: auto;
}

.c0 img {
  width: 100%;
  height: 100%;
}

<div
  class=""
  data-testid="images-layout"
>
  <div
    class="c0"
  >
    <img
      class="c1"
      src="https://images.unsplash.com/photo-1561344640-2453889cde5b?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1217&q=80"
    />
  </div>
  <div
    class="c0"
  >
    <img
      class="c1"
      src="https://images.unsplash.com/photo-1501769752-a59efa2298ce?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1050&q=80"
    />
  </div>
  <div
    class="c0"
  >
    <img
      class="c1"
      src="https://images.unsplash.com/photo-1533577116850-9cc66cad8a9b?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1050&q=80"
    />
  </div>
</div>
`;
